package.json

```diff
@@ -1,14 +1,14 @@
 {
     "name": "easystarjs",
-    "version": "0.4.3",
+    "version": "0.4.4",
     "main": "./src/easystar.js",
     "typings": "./index.d.ts",
     "description": "Asynchronous A* Pathfinding API",
     "scripts": {
-        "build": "webpack && webpack --minify",
-        "test": "./node_modules/karma/bin/karma start ./karma.conf.js",
-        "benchmark": "./node_modules/karma/bin/karma start ./karma.benchmark.conf.js",
-        "watch": "webpack -w"
+        "build:production": "webpack --production",
+        "test:production": "karma start ./karma.conf.js",
+        "benchmark": "karma start ./karma.benchmark.conf.js",
+        "dev": "webpack --development && karma start ./karma.conf.js --development"
```

Comment:
Why does dev still open karma in non-headless chrome?

Reply:
@prettymuchbryce you can debug the tests in browser, it's much more easier to debug in Chrome rather than in IDE

![image](https://user-images.githubusercontent.com/1106849/95678341-c1f58f00-0bd4-11eb-986d-d7cb0e657441.png)

![image](https://user-images.githubusercontent.com/1106849/95678363-e8b3c580-0bd4-11eb-8c2f-f1ed3c02b576.png)